FirebaseAuthUI/Sources/FUIAccountSettingsOperationDeleteAccount.m

```diff
@@ -43,7 +43,8 @@ - (void)showDeleteAccountDialog {
                                   alertMessage:FUILocalizedString(kStr_DeleteAccountBody)
                               alertCloseButton:FUILocalizedString(kStr_Cancel)
                                providerHandler:^(id<FIRUserInfo> provider) {
-    if (![provider.providerID isEqualToString:FIREmailAuthProviderID]) {
+    // TODO: Use public API after Firebase 11 minimum.
+    if (![provider.providerID isEqualToString:@"email"]) {
```

Comment:
optional: Since this is across multiple files can you define a temporary module-level constant and put the TODO over that constant? Applies to the other strings as well.

Optional since these can all be found with code search.

Reply:
Going to skip this since it would require a separate header for each product or more rearchitecture of both build systems.